lib/mp4/transmuxer.js

```diff
@@ -1213,6 +1213,7 @@ Transmuxer = function(options) {
       videoTrack.timelineStartInfo.dts = undefined;
       videoTrack.timelineStartInfo.pts = undefined;
       clearDtsInfo(videoTrack);
+      pipeline.captionStream.reset();
```

Comment:
I don't believe that's necessary, since if there's a `videoTrack` then there must be a `captionStream`